data/challenges/unlock_bonus_kart1.challenge

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<challenge version="3">
+    <!-- This challenge can only be completed
+         by dedicated code -->
+    <unlock_list list="true"/>
```

Comment:
This syntax looks weird to me? Ho does that work? I wonder if there's a way to make that clearer